app/src/main/java/org/sopt/korailtalk/core/designsystem/component/dropdown/KorailTalkBasicDropDown.kt

```diff
@@ -1,2 +1,196 @@
+    modifier: Modifier = Modifier
+) {
+    var expanded by remember { mutableStateOf(false) }
+
+    // 화살표
+    val rotateAngle by animateFloatAsState(
+        targetValue = if (expanded) 180f else 0f,
+        label = "ArrowAnimation"
+    )
+
+    Box(
+        Modifier
+            .width(68.dp)
+            .zIndex(1f)
+    ) {
+        Column(
+            Modifier
+                .width(68.dp)
+                .background(
+                    color = KorailTalkTheme.colors.white,
+                    shape = RoundedCornerShape(size = 4.dp)
+                )
+                .border(
+                    width = 1.dp,
+                    color = KorailTalkTheme.colors.gray200,
+                    shape = RoundedCornerShape(size = 4.dp)
+                )
+                .padding(bottom = if (expanded) 4.dp else 0.dp)
+        ) {
+            Row(
+                modifier = modifier
+                    .fillMaxWidth()
+                    .height(36.dp)
+                    .clickable { expanded = !expanded }
+                    .padding(start = 8.dp, top = 5.dp, end = 2.dp, bottom = 5.dp),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = selectedItem,
+                    color = KorailTalkTheme.colors.gray500,
+                    style = KorailTalkTheme.typography.body.body2M15,
+
+                    )
+                Icon(
+                    imageVector = Icons.Default.ArrowDropDown,
+                    contentDescription = "Dropdown Arrow",
+                    modifier = modifier
+                        .size(24.dp)
+                        .rotate(rotateAngle),
+                    tint = KorailTalkTheme.colors.gray500
+                )
+            }
+
+            // 펼쳐졌을 때 나오는 리스트 영역
+            if (expanded) {
+                Spacer(
+                    modifier = modifier
+                        .fillMaxWidth()
+                        .height(1.dp)
+                        .background(KorailTalkTheme.colors.gray100)
+                )
+
+                val otherItems = items.filter { it != selectedItem }
+
+                otherItems.forEachIndexed { index, item ->
+                    DropdownItem(
+                        item = item,
+                        onItemClick = {
+                            onItemSelected(item)
+                            expanded = false
+                        },
+                        isLastItem = index == otherItems.lastIndex
+                    )
+                }
+
+                Spacer(modifier = Modifier.height(4.dp))
+            }
+        }
+    }
+}
+
+@Composable
+fun DropdownItem(
+    item: String,
+    onItemClick: () -> Unit,
+    isLastItem: Boolean,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .clickable { onItemClick() }
```

Comment:
클릭 시 리플 효과를 명시해주는 것도 좋지만 패딩을 주게 되면서 클릭 영역이 명확하지 않게 된다면 차라리 리플을 아예 없애는 것도 좋아보여요. 올려주신 영상 상으로는 마지막 아이템일 때는 리플이 아이템 전체로 표시되는 게 아니라, Spacer 때문에 간격이 조금 남는 걸로 보여서요..!

처리가 어렵다면 Modifier에 리플 제거 확장함수를 미리 만들어뒀는데,
```suggestion
            .noRippleClickable(
                onClick = onItemClick
            )
```
로 바꿔 써주면 좋을 것 같습니다.